Documentation/deploy-master.md

```diff
@@ -668,6 +668,14 @@ kube-controller-$node
 kube-proxy-$node
 ```
 
+### Configure Calico
+
+To start Calico services :
+
+```
+docker run --rm --net=host -v /srv/kubernetes/manifests:/host/manifests quay.io/coreos/hyperkube:v1.5.1_coreos.0 /hyperkube kubectl apply -f /host/manifests/calico.yaml
```

Comment:
@ArchiFleKs the problem is that if a pod is started using the flannel without the Calico CNI plugin, then Calico won't know about the pod and thus won't be able to enforce policy on it, even if you restart Calico.

You'd have to restart the pod itself in order for it to be recreated with the Calico CNI plugin in order to make it work.